package.json

```diff
@@ -7,7 +7,8 @@
     "test": "jest --coverage",
     "build": "webpack -p ",
     "start:dev": "webpack -d && webpack-dev-server --mode development --open --hot",
-    "lint": "./node_modules/.bin/eslint"
+    "lint": "./node_modules/.bin/eslint",
+    "coverage": "jest --coverage"
```

Comment:
@matthenge This command will not be necessary anymore. It is similar to the command in line 7

Reply:
Okay @koechkevin let me make the necessary changes